hs_core/models.py

```diff
@@ -1642,28 +1640,46 @@ def can_be_public_or_discoverable(self):
 
         return False
 
+    # TODO: these should be instance methods, not class methods.
+    # These decisions are used when operating upon an instance.
+    # That requires less overhead than using a class method.
     @classmethod
     def get_supported_upload_file_types(cls):
```

Comment:
It seems the current use of this method is always like a class method (invoked from a class and not from an instance). Since this method is not dependent on a instance it seems it should be a class method.